Phylogenetic analysis script using NJ method with genetic distances considering nucleotide/amino acid insertions/deletions, as published in the following paper:

- T.Nishimaki, K.Sato, “An Extension of the Kimura Two-Parameter Model to the Natural Evolutionary Process.” Journal of molecular evolution 87.1 (2019): 60-67. (https://link.springer.com/article/10.1007/s00239-018-9885-1)

- T.Nishimaki, K.Sato, “P\*R\*O\*P: a web application to perform phylogenetic analysis considering the effect of gaps.” BMC Bioinformatics 22, 36 (2021). (https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-021-03978-z)


# How To Use
- Run the first cell.
- Under "Execution" cell, specify aligned sequence file name in "alignment_file".
- Set seq_type to NUCLEOTIDE or AMINO_ACID.
- Script outputs Newick format phylogenetic tree and evolutionary distance matrix.